onnxruntime/test/contrib_ops/layer_norm_op_test.cc

```diff
@@ -134,6 +130,8 @@ class LayerNormOpTester : public OpTester {
   std::vector<float> scale_data_;
   std::vector<float> B_data_;
   std::vector<float> Y_data_;
+  float mean_;
+  float inv_std_var_;
 
```

Comment:
Scott: Does anything validate that these values are calculated correctly?
Sherlock: Yes. See layer_norm_test.cc in /orttraining/test/training_ops/cuda/layer_norm_test.cc
They are only testes in training enabled case, though